grabbit/src/main/groovy/com/twcable/grabbit/ClientJobStatus.groovy

```diff
@@ -46,12 +49,22 @@ class ClientJobStatus {
         this.jcrNodesWritten = jcrNodesWritten
     }
 
-
-    public static ClientJobStatus get(@Nonnull JobExplorer explorer, @Nonnull Long jobExecutionId) {
+    /**
+     * Queries the status of a job
+     * @param explorer job explorer to query
+     * @param jobExecutionId the ID to search
+     * @throws {@link NonExistentJobException} if job can not be found. {@link IllegalArgumentException} if explorer or jobExecutionId are null
+     * @return {@link ClientJobStatus}
+     */
+    static ClientJobStatus get(@Nonnull JobExplorer explorer, @Nonnull Long jobExecutionId) {
         if (explorer == null) throw new IllegalArgumentException("JobExplorer == null")
         if (jobExecutionId == null) throw new IllegalArgumentException("JobExecutionId == null")
 
-        final jobExecution = explorer.getJobExecution(jobExecutionId)
+        final GrabbitJobExecution jobExecution = explorer.getJobExecution(jobExecutionId) as GrabbitJobExecution
```

Comment:
Trivial: `as GrabbitJobExecution` effectively declares the type to the compiler, so the type declaration is redundant.